medical_family/__openerp__.py

```diff
@@ -42,7 +42,7 @@
 {
 
     'name': 'Medical Family',
-    'version': '8.0.1.1.0',
+    'version': '9.0.1.0.0',
```

Comment:
This file should follow the same structure as [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py). The end result will be the same structure/organization as this one I keep on hand with me. All double quotes as well.
````python
{
    "name": "Odoo Medical",
    "summary": "Extends Odoo with medical patients, doctors, and centers.",
    "version": "9.0.1.0.0",
    "category": "Medical",
    "website": "https://laslabs.com",
    "author": "LasLabs, Odoo Community Association (OCA)",
    "license": "AGPL-3",
    "application": False,
    "installable": True,
    "depends": [
        "base",
        "product",
    ],
    "data": [
        "data/ir_sequence_data.xml",
        "security/medical_security.xml",
        "security/ir.model.access.csv",
        "views/medical_patient_view.xml",
        "views/res_partner_view.xml",
        "views/medical_menu.xml",
    ],
    "demo": [
        "demo/medical_patient_demo.xml",
    ],
}